<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MineSweeper</class>
 <widget class="QMainWindow" name="MineSweeper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MineSweeper</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>20</y>
      <width>54</width>
      <height>44</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
font-size:50;
color: rgb(255, 0, 0);</string>
    </property>
    <property name="digitCount">
     <number>3</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>20</y>
      <width>54</width>
      <height>44</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
font-size:50;
color: rgb(255, 0, 0);</string>
    </property>
    <property name="digitCount">
     <number>3</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/img/face_happy.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionBeginner"/>
    <addaction name="actionIntermediate"/>
    <addaction name="actionExpert"/>
    <addaction name="actionCustom"/>
   </widget>
   <addaction name="menuGame"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionBeginner">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Beginner</string>
   </property>
  </action>
  <action name="actionIntermediate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Intermediate</string>
   </property>
  </action>
  <action name="actionExpert">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Expert</string>
   </property>
  </action>
  <action name="actionCustom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Custom</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionNew</sender>
   <signal>triggered()</signal>
   <receiver>MineSweeper</receiver>
   <slot>newGame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionBeginner</sender>
   <signal>triggered()</signal>
   <receiver>MineSweeper</receiver>
   <slot>beginner()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionIntermediate</sender>
   <signal>triggered()</signal>
   <receiver>MineSweeper</receiver>
   <slot>intermediate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExpert</sender>
   <signal>triggered()</signal>
   <receiver>MineSweeper</receiver>
   <slot>expert()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCustom</sender>
   <signal>triggered()</signal>
   <receiver>MineSweeper</receiver>
   <slot>custom()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MineSweeper</receiver>
   <slot>newGame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>newGame()</slot>
  <slot>beginner()</slot>
  <slot>intermediate()</slot>
  <slot>expert()</slot>
  <slot>custom()</slot>
 </slots>
</ui>
